### PR TITLE
Make some networking and storage tests nondestructive

### PR DESCRIPTION
**Pull request by martinpitt**
See https://github.com/orgs/cockpit-project/projects/4/?pane=issue&itemId=60070873

### CODE REVIEW

**Comment by martinpitt** [comment]
Reasonably happy on our CI, but [problems on TF](https://artifacts.dev.testing-farm.io/4bebadac-b201-4fef-bc3e-181941738f78/). Doing  next round only there.

File: test/verify/check-system-info [hunk 1]
```diff
@@ -696,12 +696,14 @@ password=foobar
...
         b.logout()
 
+    @testlib.nondestructive
     def testOverview(self):
         m = self.machine
         b = self.browser
 
         # packagekit often eats a lot of CPU; silence it to not screw up the "system is idle" test
         m.execute("systemctl mask packagekit")
+        self.addCleanup(m.execute, "systemctl unmask packagekit")
```
Comment (jelly) [inline test/verify/check-system-info:706]: Woops :sweat_smile:

**Review by jelly — APPROVED** [review]
Thanks

File: test/verify/check-system-info (hunk 1 above)
Comment (martinpitt) [reply test/verify/check-system-info:706]: It was destructive before, so not a bug. But now we need to clean up after ourselves :grin:

**Pull request merged**